packages/lesswrong/components/comments/CommentOnSelection.tsx

```diff
@@ -18,6 +19,11 @@ const selectedTextToolbarStyles = (theme: ThemeType): JssStyles => ({
     "&:hover": {
       background: theme.palette.panelBackground.darken08,
     },
+
+    // Hide on mobile to avoid horizontal scrolling
+    [theme.breakpoints.down('xs')]: {
+      display: isEAForum ? "none" : "initial",
```

Comment:
Wait, does LW seriously not want this? Also, how does this relate to: https://github.com/ForumMagnum/ForumMagnum/pull/6514

Reply:
I indeed was not tracking that distinction and it is helpful